js/main.js

```diff
@@ -31,3 +31,47 @@ const appendElementWithText = function (parent, tagName, text) {
   element.appendChild(textNode)
   parent.appendChild(element)
 }
+
+const buildRecipeDetails = function (recipeDetails) {
```

Comment:
Nice function & input data structure. It's also a better abstraction now - the user doesn't care about the markup produced by the method 👍